src/main/java/edu/wpi/grip/core/sources/CameraSource.java

```diff
@@ -83,7 +94,7 @@ private synchronized void startVideo(final int deviceNumber) throws IOException
         try {
             grabber.start();
         } catch (FrameGrabber.Exception e) {
-            throw new IOException("A problem occurred trying to start the frame grabber for device number " + deviceNumber, e);
+            throw new IOException("A problem occurred trying to start the frame grabber", e);
```

Comment:
There's no common interface to get that for any frame grabber, and I think it's OK if we don't explicitly say that information